pages/edge/tutorials/cdn.mdx

```diff
@@ -0,0 +1,212 @@
+import Install from "@components/install.mdx";
+import Login from "@components/login.mdx";
+import CliVersionCallout from "@components/deploy/CliVersionCallout.mdx";
+
+# Using Wasmer Edge as a CDN
+
+In this tutorial, you will learn how to use Wasmer Edge as a CDN to host your static assets. We'll be using the [`static-web-server`](https://wasmer.io/wasmer/static-web-server)
+
+## Prerequisites
+
+The project requires the following tools to be installed on your system:
+
+- [Rust](https://www.rust-lang.org/tools/install)
+
+## Deploying a Static Site
+
+<Steps>
+
+<Install />
+
+<CliVersionCallout />
+
+<Login />
+
+### Initialize the project
+
+We'll initialize the project with the `static-site` template.
+
+Navigate to the directory where you want to create the project and run the following command:
+
+```bash
+$ wasmer app create
+```
+
+The above command will give you the following prompt:
+
+```bash
+App type:
+> Static website # Select Static website
+  HTTP server
+  Browser shell
+  JS Worker (experimental)
+  Python Application
+Who should own this package?: <your-namespace>
+No package found or specified.
+What should the package be called? It will be published under wasmer: <your-app-name> # This will be the package name of your CDN
+What should be the name of the app? <NAME>.wasmer.app: <your-app-name> # This will be the name of your app
+Would you like to publish the app now? no # Select no for now
+Writing app config to '/path/to/your/project/app.yaml'
+To (re)deploy your app, run 'wasmer deploy'
+```
+
+Select the above configuration mentioned in the comments and press enter.
+
+The above command will initialize a blank project with the following structure.
+
+<FileTree>
+  <FileTree.Folder name="your-top-level-dir" defaultOpen>
+    <FileTree.Folder name="public" defaultOpen>
+      <FileTree.File name="index.html" />
+    </FileTree.Folder>
+    <FileTree.File name="app.yaml" />
+    <FileTree.File name="wasmer.toml" />
+  </FileTree.Folder>
+</FileTree>
+
+### Changing configuration for the CDN
+
+As you might notice, this project uses the `static-web-server` package which is a static web server written in Rust.
+
+Most notably, it supports changing the compression level, cache control, cors, and more.
+All these configurations can be super useful when configuring a CDN.
+
+So, let's add a `config.toml` file in your top level directory for `static-web-server`'s configuration.
+
+<Callout type="info" emoji="ℹ️">
+  The below configuration is the configuration of the `static-web-server`
+  package. More info about it can be found
+  [here](https://static-web-server.net/configuration/config-file/).
+</Callout>
+
+You also need to update it with some changes listed below.
+
+```toml
+[general]
+
+#### Address & Root dir
+host = "::"
+port = 8080
+root = "/app/public" # 👈🏼 Change the root directory to the public directory for static assets
+
+# ... Rest of the config
+```
```

Comment:
The order of this is not very understandable.

Here's what would make it easier:

```suggestion

[The raw of the toml, I can't paste them here because Github formats it wrong]

<Callout type="info" emoji="ℹ️">
  The below configuration is the configuration of the `static-web-server`
  package. More info about it can be found
  [here](https://static-web-server.net/configuration/config-file/).
</Callout>
```

Reply:
So basically, it would render like this:

So, let's add a `config.toml` file in your top level directory for `static-web-server`'s configuration.

```toml
[general]

#### Address & Root dir
host = "::"
port = 8080
root = "/app/public" # 👈🏼 Change the root directory to the public directory for static assets

# ... Rest of the config
```

>  The below configuration is the configuration of the `static-web-server`
>  package. More info about it can be found
>  [here](https://static-web-server.net/configuration/config-file/).